datasets/ag_news/ag_news.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace NLP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""AG News topic classification dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+import nlp
+
+
+_DESCRIPTION = """\
+AG is a collection of more than 1 million news articles. News articles have been 
+gathered from more than 2000 news sources by ComeToMyHead in more than 1 year of 
+activity. ComeToMyHead is an academic news search engine which has been running 
+since July, 2004. The dataset is provided by the academic comunity for research 
+purposes in data mining (clustering, classification, etc), information retrieval 
+(ranking, search, etc), xml, data compression, data streaming, and any other 
+non-commercial activity. For more information, please refer to the link 
+http://www.di.unipi.it/~gulli/AG_corpus_of_news_articles.html .
+
+The AG's news topic classification dataset is constructed by Xiang Zhang 
+(xiang.zhang@nyu.edu) from the dataset above. It is used as a text 
+classification benchmark in the following paper: Xiang Zhang, Junbo Zhao, Yann 
+LeCun. Character-level Convolutional Networks for Text Classification. Advances 
+in Neural Information Processing Systems 28 (NIPS 2015).
+"""
+
+_CITATION = """\
+@inproceedings{Zhang2015CharacterlevelCN,
+  title={Character-level Convolutional Networks for Text Classification},
+  author={Xiang Zhang and Junbo Jake Zhao and Yann LeCun},
+  booktitle={NIPS},
+  year={2015}
+}
+"""
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/mhjabreel/CharCnn_Keras/master/data/ag_news_csv/train.csv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/mhjabreel/CharCnn_Keras/master/data/ag_news_csv/test.csv"
+
+
+class AGNewsConfig(nlp.BuilderConfig):
```

Comment:
I think this class can be removed see that it's not used? Or you can refer to the [squad](https://github.com/huggingface/nlp/blob/master/datasets/squad/squad.py) dataset on how you can use the `config class`

Reply:
thanks, I removed it!

Reply:
great, thanks